ts/Series/BoxPlot/BoxPlotSeries.ts

```diff
@@ -297,25 +352,17 @@ class BoxPlotSeries extends ColumnSeries {
 
                 // The whiskers
                 if (whiskerLength) {
-                    const whiskerStrokeWidth = point.whiskers.strokeWidth();
-                    highPlot = crisp(point.highPlot, whiskerStrokeWidth);
-                    lowPlot = crisp(point.lowPlot, whiskerStrokeWidth);
-                    pointWiskerLength = (
-                        typeof whiskerLength === 'string' &&
-                        (/%$/).test(whiskerLength)
-                    ) ?
-                        halfWidth * parseFloat(whiskerLength) / 100 :
-                        Number(whiskerLength) / 2;
-                    d = [
-                        // High whisker
-                        ['M', crisp(stemX - pointWiskerLength), highPlot],
-                        ['L', crisp(stemX + pointWiskerLength), highPlot],
+                    // Moved here as "const", refactoring of whiskerlength
+                    // triggered "no-loop-func" from eslint
```

Comment:
This comment refers to a historic state that is no longer relevant for us in the future when we want to edit this code. It's better that comments explain the code as it currently works.